Gemfile.lock

```diff
@@ -125,9 +125,12 @@ PLATFORMS
 
 DEPENDENCIES
```

Comment:
HI @aleuca ! Thanks for the PR. Please send one change per one PR. If you wish to send changes for the dependencies file, please send it in the separate PR and keep this PR for the blog post.